ibis/backends/bigquery/registry.py

```diff
@@ -688,13 +688,10 @@ def _nth_value(t, op):
     ops.RandomScalar: fixed_arity("RAND", 0),
     ops.NthValue: _nth_value,
     ops.JSONGetItem: lambda t, op: f"{t.translate(op.arg)}[{t.translate(op.index)}]",
+    ops.TimestampDiff: lambda t, op: f"TIMESTAMP_DIFF({t.translate(op.left)}, {t.translate(op.right)}, SECOND)",
```

Comment:
```suggestion
    ops.TimestampDiff: binary_infix.binary_infix_op('-'),
```
Thanks to this, the column type will be set to [INTERVAL type](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#interval_type), but i am not sure if this is the best solution.

Reply:
Oh, let me give that a try, that's better for ibis, since that's what we actually infer the type as.

Reply:
Turns out to be a big pain to do this, because the execution return type is `DateOffset`, which is pretty awful to use.

Reply:
In a follow up PR, I'll introduce a `.delta` method that allows specifying units and returning `int64`.s

Reply:
I'll rework the notebook to use the epoch seconds from the timestamps.